src/patternfly/components/FileUpload/file-upload.scss

```diff
@@ -0,0 +1,132 @@
+  --pf-c-file-upload--c-button--m-control--OutlineOffset: calc(-1 * var(--pf-global--spacer--xs));
+
+  display: flex;
+  flex-direction: column;
+
+  &.pf-m-draganddrop {
+    position: relative;
+    border: var(--pf-c-file-upload--m-draganddrop--BorderWidth) solid var(--pf-c-file-upload--m-draganddrop--BorderColor);
+
+    &::after {
+      position: absolute;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      left: 0;
+      content: "";
+      background-color: var(--pf-c-file-upload--m-draganddrop--after--BackgroundColor);
+      opacity: var(--pf-c-file-upload--m-draganddrop--after--Opacity);
+    }
+  }
+}
+
+.pf-c-input-group > .pf-c-form-control {
+  &::placeholder {
+    color: var(--pf-c-file-upload--c-input-group--c-form-control--placeholder--Color);
+  }
+}
+
+.pf-c-file-upload__text-area {
+  position: relative;
+  min-height: var(--pf-c-file-upload__text-area--MinHeight);
+  padding: var(--pf-c-file-upload__text-area--PaddingTop) var(--pf-c-file-upload__text-area--PaddingRight) var(--pf-c-file-upload__text-area--PaddingBottom) var(--pf-c-file-upload__text-area--PaddingLeft);
+  resize: vertical;
+  background-color: var(--pf-c-file-upload__text-area--BackgroundColor);
+  border: var(--pf-c-file-upload__text-area--BorderWidth) solid;
+  border-color: var(--pf-c-file-upload__text-area--BorderTopColor) var(--pf-c-file-upload__text-area--BorderRightColor) var(--pf-c-file-upload__text-area--BorderBottomColor) var(--pf-c-file-upload__text-area--BorderLeftColor);
+
+  &:not(:read-only) {
+    &.pf-m-hover,
+    &:hover {
+      --pf-c-file-upload__text-area--BorderBottomColor: var(--pf-c-file-upload__text-area--hover--BorderBottomColor);
+    }
+
+    &.pf-m-active,
+    &:active {
+      --pf-c-file-upload__text-area--BorderBottomColor: var(--pf-c-file-upload__text-area--active--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-file-upload__text-area--active--BorderBottomWidth);
+    }
+
+    &.pf-m-focus,
+    &:focus {
+      --pf-c-file-upload__text-area--BorderBottomColor: var(--pf-c-file-upload__text-area--focus--BorderBottomColor);
+
+      border-bottom-width: var(--pf-c-file-upload__text-area--focus--BorderBottomWidth);
+    }
+  }
+
+  &:read-only,
+  &.pf-m-readonly {
+    --pf-c-file-upload__text-area--BackgroundColor: var(--pf-c-file-upload__text-area--readonly--BackgroundColor);
+  }
+
+  &.pf-m-loading {
+    background-color: var(--pf-global--disabled-color--300);
+    border: 0;
+  }
+}
+
+.pf-c-button.pf-m-control {
+  --pf-c-button--PaddingRight: var(--pf-c-file-upload--c-button--m-control--PaddingRight);
+  --pf-c-button--PaddingLeft: var(--pf-c-file-upload--c-button--m-control--PaddingLeft);
+
+  outline-offset: var(--pf-c-file-upload--c-button--m-control--OutlineOffset);
+
+  &:disabled,
+  &.pf-m-disabled {
+    color: initial;
+    background-color: var(--pf-c-file-upload--c-button--m-control--disabled--BackgroundColor);
+
```

Comment:
These disabled styles, too - can we just use what's in the button component?

Reply:
the default disabled style for `pf-c-button :disabled` is below. Its very different from what @lboehling has for the disabled state. @mcoker do you mean use the variables from the top-level button disabled button component and apply them here?

<img width="327" alt="Screen Shot 2020-02-10 at 9 31 18 AM" src="https://user-images.githubusercontent.com/20118816/74158470-23013900-4be8-11ea-94f2-0107a3666ad9.png">

Reply:
Good question. I was using the sketch kit and it seems like the disabled states are treated with a gray background in split buttons/dropdowns. Is the gray bkg just pulling from the disabled form/dropdown that the action button is on? If this is just an inconsistency with the library and/or if I'm just interpreting the sketch library incorrectly, I'm fine with keeping the default disabled styling!  

what do you think? @mcarrano 
<img width="435" alt="Screen Shot 2020-02-11 at 3 57 45 PM" src="https://user-images.githubusercontent.com/46571616/74278843-d8ff7c80-4ce7-11ea-9cd1-4c8032823c0f.png">